src/types/outer/drello-api/board.d.ts

```diff
@@ -1,4 +1,7 @@
+import { Column } from "src/types/outer/column";
+
 export interface Board {
   id: number;
   title: string;
+  columns: [Column] | [];
```

Comment:
Maybe you mean this?
```suggestion
  columns: Column[] | [];
```

Reply:
Fixed it!